tests/virt/node/gpu/vgpu/conftest.py

```
@@ -76,3 +83,73 @@ def hco_cr_with_node_specific_mdev_permitted_hostdevices(
+def gpu_nodes_labeled_with_vm_vgpu(nodes_with_supported_gpus, gpu_nodes_labeled_with_vgpu_config):
+    yield from label_nodes(nodes=nodes_with_supported_gpus, labels={GPU_WORKLOAD_CONFIG_LABEL: "vm-vgpu"})
+
+
+@pytest.fixture(scope="package")
+def nvidia_vgpu_manager_ready(nvidia_vgpu_manager_ds, nodes_with_supported_gpus):
+    """Wait until nvidia-vgpu-manager DaemonSet is ready on all vGPU nodes."""
+    wait_for_nvidia_vgpu_manager(
+        nvidia_vgpu_manager_ds=nvidia_vgpu_manager_ds,
+        nodes_with_supported_gpus=nodes_with_supported_gpus,
+    )
+    yield nvidia_vgpu_manager_ds
```

Ruff (0.15.17)

[warning] No teardown in fixture `nvidia_vgpu_manager_ready`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)